Automattic/wp-calypso
# Jetpack Cloud: Disable downloadable backup creation while another one is in progress

#### Changes proposed in this Pull Request

* Refactor download flow to eliminate extraneous selectors
* Force download rewind flow screen to wait to render until the initial request for progress has finished
* Lock button if another download is in progress

<img width="2407" alt="Screen Shot 2020-07-07 at 3 51 58 PM" src="https://user-images.githubusercontent.com/2810519/86852708-17685000-c06a-11ea-835f-5063dd452a48.png">


#### Testing instructions

1. Open two different backups for the same site in side-by-side windows
2. Start downloadable file creation on one, reload the other window.
3. Verify the other button disables with "Another downloadable file is being created" until the first is complete
4. Create a fresh jetpack site ( can be JN ). Make sure the download page is able to load for a fresh backup purchase i.e. if there have been no previous downloads and the request for `/sites/:siteId/rewind/downloads` returns `[]`


Fixes 1164141197617539-as-1178122806736480
